.github/workflows/main.yml

```diff
@@ -142,4 +155,5 @@ jobs:
           cp -r $GITHUB_WORKSPACE/devops/autotest/client/ruby/l10n/bundles/* $GITHUB_WORKSPACE/l10n/bundles
           cd $GITHUB_WORKSPACE/devops/autotest/client/ruby
           sed -i 's|path => "../../../../ruby_client/singleton|path => "/home/runner/work/singleton/singleton|g' Gemfile
-          bundle exec rspec './spec/Config_spec.rb' './spec/Datetime_spec.rb' './spec/SimpleT_offline_spec.rb' './spec/SimpleT_spec .rb' './spec/Translation_offline_spec.rb' './spec/Translation_spec.rb'
\ No newline at end of file
+          bundle install
+          bundle exec rspec spec
```

Comment:
What does it mean by 'spec'?

Reply:
> 

Got confirm from QE @dleng-vmware the command should be 'bundle exec rspec spec'